### ISSUE TITLE
Add validation set?

### ISSUE THREAD

**Issue by alexpiet** [AllenInstitute/visual_behavior_glm#107]
Should we add a separate validation set? Separate from the training/test sets. I dont think this is strictly necessary, but might be a good thing to do.
